demos/benchmark/lv_demo_benchmark.h

```diff
@@ -25,6 +25,34 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
+typedef struct {
+    const char * name;
+    void (*create_cb)(void);
+    uint32_t scene_time;
+    uint32_t cpu_avg_usage;
+    uint32_t fps_avg;
+    uint32_t render_avg_time;
+    uint32_t flush_avg_time;
+    uint32_t measurement_cnt;
+} lv_benchmark_scene_dsc_t;
+
+typedef struct {
+    /*
+     * List of scenes
+     * `create_cb` == NULL indicates the last stored scene
+     * Must not be free'd
+     */
+    lv_benchmark_scene_dsc_t * scenes;
+
+    int32_t total_avg_fps;
+    int32_t total_avg_cpu;
+    int32_t total_avg_render_time;
+    int32_t total_avg_flush_time;
+    int32_t valid_scene_cnt;
+} lv_benchmark_summary_t;
+
+typedef void (*lv_on_benchmark_end_cb)(const lv_benchmark_summary_t *);
```

Comment:
Func pointer typedefs in LVGL are usually `*_cb_t`

```suggestion
typedef void (*lv_on_benchmark_end_cb_t)(const lv_benchmark_summary_t *);
```

Reply:
Thanks, fixed :D